src/main/java/db/migration/V4__incomplete_forms_default.java

```diff
@@ -0,0 +1,19 @@
+package db.migration;
+
+        import org.flywaydb.core.api.migration.spring.SpringJdbcMigration;
+        import org.springframework.jdbc.core.JdbcTemplate;
+
+/**
+ * Add default values to new incomplete forms columns
+ * @author wspride
+ */
+public class V4__incomplete_forms_default implements SpringJdbcMigration {
+
+    @Override
+    public void migrate(JdbcTemplate jdbcTemplate) throws Exception {
+        jdbcTemplate.execute("ALTER TABLE formplayer_sessions " +
+                "ALTER title SET DEFAULT 'CommCare Form'");
+        jdbcTemplate.execute("ALTER TABLE formplayer_sessions " +
+                "ALTER dateOpened SET DEFAULT 'Tue Jul 3 14:19:24 EDT 1990'");
```

Comment:
this is kind of a confusing default, can we set it to either `CURRENT_DATE` or Jan 1 1970 (beginning of epoch)?